src/tracer.cpp

```diff
@@ -110,14 +110,20 @@ std::unique_ptr<ot::Span> Tracer::StartSpanWithOptions(ot::string_view operation
                                       opts_.service, opts_.type, operation_name, operation_name,
                                       opts_.operation_name_override}};
   bool is_trace_root = parent_id == 0;
+  if (!opts_.version.empty()) {
```

Comment:
for other languages we implemented as applying `DD_TAGS` first, and then adding `DD_VERSION` and `DD_ENV`.

the thinking was to encourage the use of `DD_VERSION` instead of `DD_TAGS=version:1.2.3,env:my-env`.

for example, given:

```
DD_VERSION=1.2.3 DD_TAGS=version:4.5.6 ./bin/my-app
```

Would result in:

```json
{
  "meta": {
    "version": "1.2.3"
   }
}
```

This is not a hard requirement set anywhere, was just the way we approached in other places.

Reply:
(which is seems like we do for `env` below too)